doc/mockups/budget-reporting.txt

```diff
@@ -12,7 +12,7 @@ Change of balance (flow):
 $ hledger bal ^expenses -M --depth 1 --budget-simple  # assume a fixed budget
 Change of balance (flow, with simple budget):
 
-          ||          | 2014/01/01-2014/01/31  2014/02/01-2014/02/28  2014/03/01-2014/03/31  |   average     total
+          ||          | 2014-01-01-2014-01/31  2014-02-01-2014-02-28  2014-03-01-2014-03-31  |   average     total
```

Comment:
looks like one overlooked slash fwiw

Reply:
Thanks, @danopia . I just pushed the fix for this.